include/odp/api/spec/packet_io.h

```diff
@@ -673,6 +675,25 @@ typedef struct odp_pktio_capability_t {
 	/** Packet input vector capability */
 	odp_pktin_vector_capability_t vector;
 
+	/** Supported frame lengths for odp_pktio_maxlen_set()
+	 *
+	 * A frame length value of zero indicates an unsupported operation. If
+	 * the 'common_len' flag is set, the same maximum frame length is used
+	 * for both input and output directions. In this case the min/max values
+	 * for input and output are identical. */
+	struct {
+		/** Common maximum frame length for both packet input and output */
+		odp_bool_t common_len;
+		/** Minimum supported frame length at packet input */
+		uint32_t min_input_len;
+		/** Maximum supported frame length at packet input */
+		uint32_t max_input_len;
+		/** Minimum supported frame length at packet output */
+		uint32_t min_output_len;
+		/** Maximum supported frame length at packet output */
+		uint32_t max_output_len;
+	} max_frame_len;
```

Comment:
Naming and documentation could be clearer that these are min/max valid values for odp_pktio_maxlen_set() params (and not the current link min/max frame sizes).

Reply:
Ack